test/unittests/common/commonFormat_test.cpp

```diff
@@ -31,15 +31,15 @@
 
```

Comment:
I thougth that all the test/unittests/ directories were covered... However, this PR include fixes in one of them. Why the existing checks have not detected this? Maybe the directory has not been included in the style check script list?